andela/hc-the-knights
# #153728033 Add nagging email feature when check goes down

#153728033 Add a feature to nag user via email when checks goes down until he resolves issues he has been notified about
#### What does this PR do?
Nags a user via email when a check goes down until he resolves issues he has been notified about
#### Description of Task to be completed?
The user should be nagged by the system until he resolve issues he have been notified about
#### How should this be manually tested?
After cloning the repo, navigate using cd in to the directory and run 
``pip install -r hc-the-knights/requirements.txt``
create database "hc"
`` $ psql --user postgres
  postgres=# create database hc;``
create database tables, triggers and superuser
``$./manage.py migrate
$./manage.py ensuretriggers
$./manage.py createsuperuser``
then run the development server
``$./manage.py runserver``
The site should be running at ``http://localhost:8080``
Visit the site and enter your working email and check a link send to you mail for login
Add your checks, give it a name and setup the time period and grace to your liking
Copy the ping URL and past in new tab of your browser to activate your checks with a green tick in a circle.
Your will receive an email with check up status and after the expiry of the time period and grace period, you will receive another email with check down status which will continue every after period + grace time lapses until you resolve the issue back to up.
#### What are the relevant pivotal tracker stories?
#153728033
